### PR TITLE
Lots of bardo

### PR DESCRIPTION
**Pull request by tomasdisk**
### Lots of bardo
* add TS support
* add MDX component
* add git videos
* run prettier and eslint
* update Author and add images
* add GH action for lint

### CODE REVIEW

**Comment by emalorenzo** [comment]
#### 📝Changed routes:


| [**…demy/\[module\]/\[lesson\]**](https://coderhood-kmzyhbau1-coderhood.vercel.app/academy/[module]/[lesson]) | [**/academy/\[module\]**](https://coderhood-kmzyhbau1-coderhood.vercel.app/academy/[module]) |
|:-:|:-:|
|<a href="https://coderhood-kmzyhbau1-coderhood.vercel.app/academy/[module]/[lesson]">Screenshot of /academy/[module]/[lesson]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/rHsDc9XGcJOmyMR6cBUZLmdLwzTDTe7-o28TZwY1_h_jLPojyyFiRhYmFUqxMxnFWM4TnEqgSrdhXQP6pbgPUw.png">(view full size)</a> |<a href="https://coderhood-kmzyhbau1-coderhood.vercel.app/academy/[module]">Screenshot of /academy/[module]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Hek0hMIb_anrqAm3DcmgORSmPp5zMvLce-3RYL6PEBcZ7dEWy_2EWE1B4jAiEx1hEP-QRlauxX8NZ86rkxmOVg.png">(view full size)</a> |



| [**/**](https://coderhood-kmzyhbau1-coderhood.vercel.app/) | [**/profile**](https://coderhood-kmzyhbau1-coderhood.vercel.app/profile) |
|:-:|:-:|
|<a href="https://coderhood-kmzyhbau1-coderhood.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/aw2X_soVa2ZnY7UNgFjJm6u-wMa89hdrE1uUMaC2JFKV47WNAJ7Y0bneKj0Hy5bl4MWzqpQ7Wmx3HZxFmY4ayw.png">(view full size)</a> |<a href="https://coderhood-kmzyhbau1-coderhood.vercel.app/profile">Screenshot of /profile</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/KaY6tY2fVBEQw674_w_cLRlGt-6ywfGeVmsuTAMpSbFhhY5ZSWkLgwkmKLGa10bwtZT6xyevW8gqNDkE_j4rew.png">(view full size)</a> |







Commit 405e3947f57d7151ddfeaf33573aaf5c69e08cc4 (https://coderhood-kmzyhbau1-coderhood.vercel.app).

**Comment by tomasdisk** [comment]
@emaLorenzo no dejes pasar este PR, no son cambios grandes.. son varias cosas chiquitas xD

**Comment by emalorenzo** [comment]
Ni lo había visto! Jaja ahí me salió la noti. Termino una reunión y lo reviso ❤️> @emaLorenzo no dejes pasar este PR, no son cambios grandes.. son varias cosas chiquitas xD

File: src/components/MDX.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const MDX: React.FC<{ source: MDXRemoteSerializeResult<Record<string, unknown>> }> = ({
+  source,
+}) => {
+  return (
+    <MDXRemote
+      {...source}
+      components={{
+        Steps: ({ children }) => <ul>{children}</ul>,
+        Image: ({ alt, ...props }) => <img alt={alt} {...props} />,
+      }}
```
Comment (tomasdisk) [inline src/components/MDX.tsx:13]: allow us to start styling and customizing deeply our MDX in a single place 😄

**Comment by tomasdisk** [comment]
🐳 💨

File: src/components/MDX.tsx (hunk 1 above)
Comment (emalorenzo) [reply src/components/MDX.tsx:13]: de unaa

**Review by emalorenzo — APPROVED** [review]

**Pull request merged**